e2e/utils/TestUtils.ts

```diff
@@ -254,22 +254,32 @@ export async function tool(tool: string, ...args: string[]): Promise<string> {
   }
 }
 
-export async function waitForRestartVM(progressBar: Locator, options = { timeout: 10_000, interval: 200 }): Promise<void> {
+export async function waitForRestartVM(progressBar: Locator, options = { timeout: -1, interval: 200 }): Promise<void> {
   const startTime = new Date().valueOf();
-  const endTime = startTime + options.timeout;
+  let endTime = startTime + options.timeout;
```

Comment:
With the new default for `timeout`, this sets `endTime` to one ms before `nowTime` at first. This isn't causing any problem, but it is bad practice IMO.

Reply:
Good catch. Fixed by dropping the `options` argument and setting `timeout` before this line.